src/Http/Http.Extensions/src/QueryBuilder.cs

```diff
@@ -23,6 +25,12 @@ public QueryBuilder(IEnumerable<KeyValuePair<string, string>> parameters)
             _params = new List<KeyValuePair<string, string>>(parameters);
         }
 
+        public QueryBuilder(IEnumerable<KeyValuePair<string, StringValues>> parameters)
+            : this(parameters.SelectMany(kvp => kvp.Value, (kvp, v) => KeyValuePair.Create(kvp.Key, v.ToString())))
```

Comment:
I think `v` is already a string, so calling `ToString()` on it isn't necessary.
```suggestion
            : this(parameters.SelectMany(kvp => kvp.Value, (kvp, v) => KeyValuePair.Create(kvp.Key, v)))
```